.github/workflows/build-test.yml

```diff
@@ -280,14 +280,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        jupyter: [raw, local]
-        python: [nonConda, conda]
-        pythonVersion: ['3.9', '3.10']
+        jupyter: [raw]
```

Comment:
Its upto maintainers of the code to have different test matrices for each area
We only run all tests in zmq and run specific tests in remote, jupyter, conda, etc